client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -59,26 +63,37 @@ const EmailProvidersStackedComparison = ( {
 	domainsWithForwards,
 	isGSuiteSupported,
 	selectedDomainName,
+	selectedEmailProviderSlug,
+	selectedIntervalLength = IntervalLength.ANNUALLY,
 	selectedSite,
 	showEmailForwardLink = true,
 	siteName,
 	source,
 }: EmailProvidersStackedComparisonProps ): ReactElement => {
 	const translate = useTranslate();
 
-	const [ intervalLength, setIntervalLengthPure ] = useState( IntervalLength.ANNUALLY );
+	const [ detailsExpanded, setDetailsExpanded ] = useState( () => {
+		if ( selectedEmailProviderSlug === 'google-workspace' ) {
+			return {
+				titan: false,
+				google: true,
+			};
+		}
```

Comment:
Do we want to check the interval length? This logic would expand Google for the monthly view, even if it's not supported and the card is not displayed.
I don't think this is blocking, as we're likely going to disable the button if that's the case.

Reply:
Yes, that was my thought. I didn't want to make that logic more complex given it's not going to be used in the near future (unless we decide to show a placeholder for Google Workspace monthly in the mean time).